CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Fixes
+
+- The SDK no longer creates warnings when loading the options ([#1128](https://github.com/getsentry/sentry-unity/pull/1128))
```

Comment:
I think this should get a more prominent entry - it is a breaking change and users need to actively migrate their runtime options configs.

Reply:
I thought so too and wondered if that should happen during release. But you're right. I added the fix as breaking change "changelog.. type?" (which I took from the Cocoa SDK)